### DISCUSSION TITLE
Hi everyone, I don't know what to do

### DISCUSSION THREAD

**Discussion by comtoplay** [symfony/symfony#54019] [Q&A]
Hi,
I can't create a new Symfony 7 project webapp
Still the same problem
`symfony new demo --version="7.0.*" --webapp `
* Creating a new Symfony 7.0.* project with Composer
  WARNING: Unable to find Composer, downloading one. It is recommended to install Composer yourself at https://getcomposer.org/download/
  (running C:\Users\cpesa\.symfony5\composer\composer.phar create-project symfony/skeleton Z:\symfony7\demo 7.0.* --no-interaction)
  
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking symfony/flex (v2.4.4)
  
  I deleted composer.phar to be sure it will downloaded from the initial installation.
But it makes a Symfony 5 folder, not good I guess
C:\Users\cpesa\**.symfony5**\composer\composer.phar create-project symfony/skeleton Z:\symfony7\demo 7.0.* --no-interaction
  
  I'm running on
  Windows 11 Pro
  Docker is installed
  Composer is up-to-date
  Symfony CLI too
  Symfony 5 and 6 projects work

What do You think?

This is my environment
  
  **symfony book:check-requirements**
[OK] Git installed
[OK] PHP installed version 8.2.12 (C:\xampp8.2.12\php\php.exe)
[OK] PHP extension "xsl" installed - required
[OK] PHP extension "sodium" installed - required
[OK] PHP extension "zip" installed - optional - needed only for chapter 17 (Panther)
[OK] PHP extension "session" installed - required
[OK] PHP extension "ctype" installed - required
[OK] PHP extension "tokenizer" installed - required
[OK] PHP extension "amqp" not found, optional - needed only for chapter 32
[OK] PHP extension "intl" installed - required
[OK] PHP extension "curl" installed - optional - needed only for chapter 17 (Panther)
[OK] PHP extension "gd" installed - optional - needed only for chapter 23 (Imagine)
[OK] PHP extension "redis" not found, optional - needed only for chapter 31
[OK] PHP extension "json" installed - required
[OK] PHP extension "pdo_pgsql" installed - required
[OK] PHP extension "mbstring" installed - required
[OK] PHP extension "xml" installed - required
[OK] PHP extension "openssl" installed - required
[OK] Composer installed
[OK] Docker installed
[OK] Docker Compose installed
[OK] npm installed

**symfony version**
Symfony CLI version 5.8.11 (c) 2021-2024 Fabien Potencier (2024-02-15T14:20:36Z - stable)
**php --version**
PHP 8.2.12 (cli) (built: Oct 24 2023 21:15:15) (ZTS Visual C++ 2019 x64)
Copyright (c) The PHP Group
Zend Engine v4.2.12, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.12, Copyright (c), by Zend Technologies
**composer --version**
Composer version 2.7.1 2024-02-09 15:26:28
    
Thank you !!!

**Comment by mbabker** [comment]
That `.symfony5` folder is for the Symfony CLI tool, which has a different version than the PHP framework.  So that's not an issue.

**Reply by comtoplay** [reply]
Hi, Thank you so much for your promt answer. I've found the reason. I was trying to create on a NAS with a web server running on it. When writing to the NAS's web directory, it was indexing at the same time as it was writing, which caused sporadic errors. Without changing anything in my configuration, I tried writing to the root of my internal disk on the development machine, and everything works perfectly. Thank you very much again. Looking forward to you!!

**Comment by comtoplay** [comment]
As I just said, I've found why I coudn't create projects. YOU DON'T HAVE TO WORK ON A NAS running web services ://
Thanks everybody!
